web-client/public/js/graph.js

```diff
@@ -43,7 +43,8 @@ export var drawGraph = function (network, sliderController, nodeColoring) {
         return node.name.length * 12 + 5;
     };
 
-  // If colorOptimal is false, then weighting is ignored, and the lines are all drawn as if it was an unweighted sheet
+    /* If colorOptimal is false, then weighting is ignored,
+    and the lines are all drawn as if it was an unweighted sheet */
     if (!$("#colorEdges").hasClass("active")) {
         colorOptimal = false;
     }
```

Comment:
Great; now that this is restored, looking forward to the next round you can think about what exactly is the right way to handle this. Because of the directionality of the MVC cycle, this code will not translate directly. Before I just tell you what it should translate to, try your hand at it and I can verify. It is definitely not a copy-paste, but it involves similar parts (plus possibly one more that isn't in the above 3 lines).